docs/content/guides/dagster/ml-ops.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Building your machine learning pipelines In Dagster | Dagster Docs
+description: This guide illustrates how to use Dagster to operationaize your machine learning pipeline
+---
+
+# Building your machine learning pipelines with Dagster
+
+In this guide, we’ll walk you through how to take your machine learning models and deploy and mantain them in production using Dagster, reliably and efficently.
+
+<Note>
+  This guide assumes you have familiarity with machine learning concepts,{" "}
+  <a href="/tutorial">the Dagster tutorial</a> and several Dagster concepts,
```

Comment:
I don't think we necessarily assume they have familiarity with the tutorial, just with the concepts. We can suggest that they take the tutorial if they're not familiar with the concepts.

Reply:
As this guide builds on the tutorial, that's why there's familiarity suggested here. I agree we can rephrase though. Perhaps something along these lines?:

> This guide assumes you have familiarity with machine learning and several Dagster concepts. It also builds on the [asset tutorial](), which we recommend completing if you're unfamiliar with Dagster or want a comprehensive look at the project code.

Reply:
+1